acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/privacy/PrivacyGroupAcceptanceTest.java

```diff
@@ -103,6 +103,66 @@ public void nodeCanCreatePrivacyGroup() {
     assertThat(privacyGroups.get(0).getMembers().length).isEqualTo(2);
   }
 
+  @Test
+  public void nodeCanCreatePrivacyGroupWithoutName() {
+    final String privacyGroupId =
+        privacyNet
+            .getNode("Alice")
+            .execute(
+                privateTransactions.createPrivacyGroupWithoutName(
+                    List.of(
+                        privacyNet.getEnclave("Alice").getPublicKeys().get(0),
+                        privacyNet.getEnclave("Bob").getPublicKeys().get(0)),
+                    "my group description"));
+
+    assertThat(privacyGroupId).isNotNull();
+
+    final List<PrivacyGroup> privacyGroups =
+        privacyNet
+            .getNode("Alice")
+            .execute(
+                privateTransactions.findPrivacyGroup(
+                    List.of(
+                        privacyNet.getEnclave("Alice").getPublicKeys().get(0),
+                        privacyNet.getEnclave("Bob").getPublicKeys().get(0))));
+
+    assertThat(privacyGroups.size()).isEqualTo(1);
+    assertThat(privacyGroups.get(0).getPrivacyGroupId()).isEqualTo(privacyGroupId);
+    assertThat(privacyGroups.get(0).getName()).isEqualTo("Default Name");
```

Comment:
Testing that "Default Name" comes in here is _really_ testing Orion functionality - not Pantheon.
Secondly - if an arbitrary name is going to be applied to the group, I'd argue that a blank string may be more appropriate.

Reply:
agree on the blank string defaults https://pegasys1.atlassian.net/browse/PAN-3098